src/Club/club.web.controller.ts

```diff
@@ -0,0 +1,296 @@
+import { UseGuards } from '@nestjs/common';
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpCode,
+  Post,
+  Put,
+  Query,
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import {
+  ApiBearerAuth,
+  ApiOperation,
+  ApiQuery,
+  ApiResponse,
+  ApiTags,
+} from '@nestjs/swagger';
+import { User } from 'src/auth/decorators';
+import { ClubService } from './club.service';
+import { acceptUserDto } from './dto/accept.dto';
+import { clubDatadto } from './dto/clubData.dto';
+import { createClubDto } from './dto/createClub.dto';
+import { editClubdto } from './dto/editclub.dto';
+
+@ApiTags('CLUB')
+@Controller('club/web')
+export class ClubController {
+  constructor(private clubService: ClubService) {}
+  @ApiBearerAuth('access-token')
+  @ApiOperation({
+    summary: '동아리 리스트 가져오기',
+    description: '동아리들을 가져옵니다',
+  })
+  @ApiQuery({
+    name: 'type',
+    description: '동아리 타입',
+    enum: ['MAJOR', 'FREEDOM', 'EDITORIAL'],
+  })
+  @ApiResponse({ status: 200, description: '동아리들 가져옵니다' })
+  @UseGuards(AuthGuard('jwt-web'))
+  @Get('/web/list')
```

Comment:
도메인이 `club/web/web/list` ??

```suggestion
  @Get('/list')
```